test/integration/multiplexed_integration_test.h

```diff
@@ -16,10 +16,10 @@ class Http2IntegrationTest : public HttpProtocolIntegrationTest {
   void simultaneousRequest(int32_t request1_bytes, int32_t request2_bytes);
 
 protected:
-  // Utility function to add filters.
-  void addFilters(std::vector<std::string> filters) {
+  // Utility function to prepend filters.
```

Comment:
nit: I guess we could mention here that the filters are going to be prepended in reverse order.